cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -48,178 +39,51 @@ Then('{string} radioButton on preview is {string}', (position, text) => {
+    .and('not.be.disabled')
+    .and('not.have.attr', 'disabled');
 });
 
-Then('{string} RadioButton is set to reverse', (position) => {
-  switch (position) {
-    case 'First':
-      radioButtonComponent(FIRST_RADIOBUTTON).children().children()
-        .find(`div:nth-child(${FIRST_ELEMENT})`)
-        .should('have.css', 'box-sizing', 'border-box');
-      break;
-    case 'Second':
-      radioButtonComponent(SECOND_RADIOBUTTON).children().children()
-        .find(`div:nth-child(${FIRST_ELEMENT})`)
-        .should('have.css', 'box-sizing', 'border-box');
-      break;
-    case 'Third':
-      radioButtonComponent(FIRST_RADIOBUTTON).children().children()
-        .find(`div:nth-child(${FIRST_ELEMENT})`)
-        .should('have.css', 'box-sizing', 'border-box');
-      break;
-    default: throw new Error('There are only three radio elements on the page');
-  }
+Then('{string} RadioButton is set to reverse', (radioButtonName) => {
+  cy.radioButton(radioButtonName, 'div:nth-child(2) input').should('have.attr', 'role', 'radio');
 });
 
-Then('{string} RadioButton is not set to reverse', (position) => {
-  switch (position) {
-    case 'First':
-      radioButtonComponent(FIRST_RADIOBUTTON).children().children()
-        .find(`div:nth-child(${FIRST_ELEMENT})`)
-        .should('have.css', 'box-sizing', 'border-box');
-      break;
-    case 'Second':
-      radioButtonComponent(SECOND_RADIOBUTTON).children().children()
-        .find(`div:nth-child(${FIRST_ELEMENT})`)
-        .should('have.css', 'box-sizing', 'border-box');
-      break;
-    case 'Third':
-      radioButtonComponent(FIRST_RADIOBUTTON).children().children()
-        .find(`div:nth-child(${FIRST_ELEMENT})`)
-        .should('have.css', 'box-sizing', 'border-box');
-      break;
-    default: throw new Error('There are only three radio elements on the page');
-  }
+Then('{string} RadioButton is not set to reverse', (radioButtonName) => {
+  cy.radioButton(radioButtonName, 'div:nth-child(1) input').should('have.attr', 'role', 'radio');
 });
 
-Then('{string} RadioButton size on preview is set to {string}', (position, size) => {
+Then('{string} RadioButton size on preview is set to {string}', (radioButtonName, size) => {
   if (size === 'small') {
-    switch (position) {
-      case 'First':
-        radioButtonByPosition(FIRST_RADIOBUTTON).should('have.css', 'width', '16px')
-          .and('have.css', 'height', '16px');
-        break;
-      case 'Second':
-        radioButtonByPosition(SECOND_RADIOBUTTON).should('have.css', 'width', '16px')
-          .and('have.css', 'height', '16px');
-        break;
-      case 'Third':
-        radioButtonByPosition(THIRD_RADIOBUTTON).should('have.css', 'width', '16px')
-          .and('have.css', 'height', '16px');
-        break;
-      default: throw new Error('There are only three radio elements on the page');
-    }
+    cy.radioButton(radioButtonName, 'div:nth-child(1) input').should('have.css', 'width', '16px')
```

Comment:
could we pass those values thought parameters?

Reply:
fixed